components/link/basic/src/index.js

```diff
@@ -25,49 +27,64 @@ export default function LinkBasic({
   useReactRouterLinks,
   url = '#'
 }) {
-  const linkClassName = cx('sui-LinkBasic', className)
+  const BASE_CLASS = 'sui-LinkBasic'
+  const linkBasicClassName = cx(BASE_CLASS, className)
```

Comment:
`@s-ui/component-dependencies`is no longer including classnames package. Plz add the 2.2.5 version to the dependencies.